tests/integration/golang/fixtures/run.go

```diff
@@ -74,11 +76,14 @@ func (f RunFixtures) CreateTestRuns(
 // GetTestRuns fetches all runs for an experiment
 func (f RunFixtures) GetTestRuns(
 	ctx context.Context, experimentID int32) ([]models.Run, error) {
-	runs := []models.Run{}
-	if err := f.db.WithContext(ctx).
-		Where("experiment_id = ?", experimentID).
-		Order("start_time desc").
-		Find(&runs).Error; err != nil {
+	var runs []models.Run
```

Comment:
I think the original layout is much more readable. Are you following some particular style guide I should know about?

Reply:
not really. I just started with this layout probably everywhere. We can switch to you layout - not a problem, but in that case we have to probably change in alllllll places which is time consuming process. that's why I decided to do vice versa right now.